docs/how-to-open-a-pull-request.md

```diff
@@ -92,61 +92,61 @@ When you are working on regular bugs and features on our development branch `mas
 
 1. Rebase your local copy:
 
-   ```console
-   git checkout <pr-branch>
-   git pull --rebase upstream master
-   ```
+    ```console
+    git checkout <pr-branch>
+    git pull --rebase upstream master
+    ```
 
 2. Resolve any conflicts and add / edit commits
 
-   ```console
-   # Either
-   git add .
-   git commit -m "chore: resolve conflicts"
+    ```console
+    # Either
+    git add .
+    git commit -m "chore: resolve conflicts"
 
-   # Or
-   git add .
-   git commit --amend --no-edit
-   ```
+    # Or
+    git add .
+    git commit --amend --no-edit
+    ```
 
 3. Push back your changes to the PR
 
-   ```console
-   git push --force origin <pr-branch>
-   ```
+    ```console
+    git push --force origin <pr-branch>
+    ```
 
 ### For upcoming curriculum and features
 
 When you are working on features for our upcoming curriculum `next-*` branches, you have do a cherry pick:
 
 1. Make sure your upstream comes in sync with your local:
 
-   ```console
-   git checkout master
-   git fetch --all --prune
-   git checkout next-python-projects
-   git reset --hard upstream/next-python-projects
-   ```
+    ```console
+    git checkout master
+    git fetch --all --prune
+    git checkout next-python-projects
+    git reset --hard upstream/next-python-projects
+    ```
 
 2. Take backup
 
-   a. Either delete your local branch after taking a backup (if you still have it locally):
+  a. Either delete your local branch after taking a backup (if you still have it locally):
 
       ```console
-      git checkout <pr-branch-name>
+        git checkout <pr-branch-name>
 
-      # example:
-      # git checkout feat/add-numpy-video-question
+        # example:
+        # git checkout feat/add-numpy-video-question
 
-      git checkout -b <backup-branch-name>
+        git checkout -b <backup-branch-name>
 
-      # example:
-      #  git checkout -b backup-feat/add-numpy-video-question
```

Comment:
I don't know why, but this is not rendering properly

Reply:
This appears to render as expected now.